bin/app.rb

```diff
@@ -1,5 +1,40 @@
-require_relative '../application_config'
-
+require_relative '../boot'
 class App
   include AwsLoader
+
+  OPERATIONS = ['create', 'delete', 'list']
+
+  def execute_operation(options)
+    operation = options.delete_at(0)
+    bucket_name = options[0]
+    
+    case operation
+    when 'create'
+      # Store.validate(options[0])
+      store = Store.new
+      store.create_bucket(bucket_name)
+    when 'delete'
+      s3_bucket = Bucket.new(bucket_name)
+      s3_bucket.delete_bucket
+    when 'list'
+      store = Store.new
+      store.list_buckets
+    else
+      raise S3StoreNoServiceError, "Invalid operation name. Try #{OPERATIONS.join(', ')}.".colorize(:red) if (operation.nil? || !OPERATIONS.include?(operation.downcase))
```

Comment:
Use `exclude?` instead of `!OPERATIONS.include?(...)`

Reply:
`exclude` is part of Rails. not pure Ruby.

Reply:
Yes that is correct - thanks @rocky02